src/jvm/io/fsq/spindle/runtime/DateFieldHelpers.scala

```diff
@@ -17,4 +17,11 @@ object DateFieldHelpers {
     javaUtilDateFormatter.parseDateTime(dateString).toDate
   }
 
+  def printLocalDate(date: org.joda.time.LocalDate): String = {
```

Comment:
hm can you `import org.joda.time.LocalDate` above? we try to keep away from fully qualified names in-line because they can confuse buildgen

that probably doesn't make a difference here, but worth being consistent